xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-security/xwiki-platform-extension-security-index/src/main/java/org/xwiki/extension/security/internal/DefaultReviewsFetcher.java

```diff
@@ -62,25 +61,19 @@ public class DefaultReviewsFetcher implements ReviewsFetcher
     @Inject
     private Logger logger;
 
+    @Inject
+    private HttpClientFactory httpClientFactory;
```

Comment:
We should introduce a new, more generic, factory (this one is quite ExtensionManager centric) but that should do it for now.

Reply:
done